docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+
+.. code-block::
+
+    Error executing command: could not read migrations in 
+    myproject/dbschema/migrations:
+
+    could not read migration file myproject/dbschema/migrations/00002.edgeql:
+
+    Migration name should be:
+    m13g7j2tqu23yaffv6wkn2adp6hayp76su2qtg2lutdh3mmj5xyk6q, but
+    m1xseswmheqzxutr55cu66ko4oracannpddujg7gkna2zsjpqm2g3a found instead.
+
+
+    Migration names are computed from the hash of the migration contents.
+
+    To proceed you must fix the statement to read as:
+    CREATE MIGRATION m13g7j2tqu23yaffv6wkn2adp6hayp76su2qtg2lutdh3mmj5xyk6q 
+    ONTO ...
+    Alternatively, revert the changes to the file.
+
+You can also create a migration that combines schema changes and data. 
+This is even easier, since it doesn't even require appending ``allow-empty`` 
+to the command. Just do the following:
+
+- Change your schema,
+- Type ``edgedb migration create`` and respond to the CLI's questions,
+- Add your queries to the file (best done on the bottom after the 
+  DDL statements have changed the schema),
+- Type ``edgedb migrate`` and change the migration name to the suggested name,
+- Type ``edgedb migrate`` again.
+
+The `EdgeDB tutorial <tutorial_>`_ is a good example of a database 
+set up with both a schema migration and a data migration. Setting 
+up a database with schema changes in one file and default data in 
+a second file is a nice way to separate the two operations and maintain 
+high readability at the same time. These two files can be `seen here 
+<tutorial_files_>`_.
+
+Squashing migrations
+====================
+
+Users often end up making frequent changes to their schema because 
+of how effortless it is to do. (And in the next section we will learn 
+about EdgeDB Watch, which is ever more effortless!) This leads to 
+an interesting side effect: lots of ``.edgeql`` files, many of which 
+represent trials and approaches that don't end up making it to the 
+final form for your schema.
+
+Once you are done, you might want to squash the migrations into a 
+single file. This is especially nice if you need to frequently initialize 
+database instances using the same schema, because all migrations are 
+applied when an instance starts up. You can imagine that the output 
+would be pretty long if you had dozens and dozens of migration files 
+to work through:
+
+.. code-block::
+
+    Initializing EdgeDB instance...
+    Applying migrations...
+    Applied m13brvdizqpva6icpcvmsc3fee2yt5j267uba6jugy6iugcbs2djkq 
+    (00001.edgeql)
+    Applied m1aildofb3gvhv3jaa5vjlre4pe26locxevqok4semmlgqwu3xayaa 
+    (00002.edgeql)
+    Applied m1ixxlsdgrlinfijnrbmxdicmpfav33snidudqi7fu4yfhg4nngoza 
+    (00003.edgeql)
+    Applied m1tsi4amrdbcfjypu72duyckrlvvyb46r3wybd7qnbmem4rjvnbcla 
+    (00004.edgeql)
+    ...and so on...
+    Project initialized.
+
+To squash your migrations, just add ``--squash`` after ``edgedb migration 
+create``. Running this command will first display some helpful info 
```

Comment:
```suggestion
To squash your migrations, just run ``edgedb migration create`` with the
 ``--squash`` option. Running this command will first display some helpful info 
```
"just add…" implies the user would have already been running `edgedb migration create`, but in this instance, the user would only run it if their intention was to squash since there are no outstanding schema changes to create migrations from. Reformulated to reflect this.